dotcom-rendering/src/paletteDeclarations.ts

```diff
@@ -3849,6 +3854,9 @@ const liveBlockBorderBottomDark: PaletteFunction = () =>
 const subMetaLabelTextLight: PaletteFunction = ({ theme, design }) => {
 	switch (theme) {
 		case ArticleSpecial.Labs:
+			if (design === ArticleDesign.Gallery) {
+				return sourcePalette.neutral[60];
```

Comment:
Are we entirely sure this is the correct color in light mode for "Explore more on these topics"?

Reply:
Nice, this is correct 👍